src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -1110,6 +1180,39 @@ export class IssueReporter extends Disposable {
 		}
 	}
 
+	private setLoading() {
+		// Show loading
+		this.previewButton.label = 'Loading Extension Data...';
```

Comment:
I don't think you need to mess with this label. The fact that it's disabled and it shows a spinner is enough

Reply:
actually, if you look at the impl of `this.updatePreviewButtonState();` it handles a lot of this state. I think it might be best to edit `this.updatePreviewButtonState();` to handle this case better instead of handling the enable state in this function.